### PR TITLE
fixes https://github.com/ssl-hep/X509_Secrets/issues/10

### PR DESCRIPTION
**Pull request by ivukotic**
original code was never actually refreshing proxy file, it was also trying to recreate secret that was already there and failing, so pod would restart every 6h.

### CODE REVIEW

**Comment by gordonwatts** [comment]
This looks like it had major changes to the setup, the OSG version, etc. Has it been tested running?

**Comment by gordonwatts** [comment]
I also added @BenGalewsky to the list as I think he was one of the original authors for these files.

**Comment by ivukotic** [comment]
Hi,

I have tested it, but one more person doing a test is always good to have.
I updated OSG version. This version already has python3.6 so I cleaned up the image.
Also I changed the code remaking the secret.

Cheers,
Ilija

**Comment by BenGalewsky** [comment]
I'll take a look at it this afternoon. Was waiting for some stability on River, but this looks like a good step forward for the whole stack.

File: Dockerfile [hunk 1]
```diff
@@ -47,5 +30,5 @@ RUN echo "Timestamp:" `date --utc` | tee /image-build-info.txt
 
 ENV X509_USER_PROXY /etc/grid-security/x509up
 
-CMD sh -c "python x509_updater.py --voms $VOMS"
+CMD sh -c "python3 x509_updater.py --voms $VOMS"
```
Comment (BenGalewsky) [inline Dockerfile:33]: This `CMD` is ignored in the helm chart and is set in the x509 deployment yaml. This PR will depend on a change in that chart

**Comment by BenGalewsky** [comment]
Looks good. Requires [ServiceX PR 248](https://github.com/ssl-hep/ServiceX/pull/248) to work

**Pull request merged**